spec/models/survey_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```

Comment:
These files will fail, since we don't have testing yet, I would just change it to

`describe Survey do
end`